design/resources.go

```diff
@@ -94,6 +94,29 @@ var _ = Resource("workitem", func() {
 
 })
 
+// new version of "list" for migration
+var _ = Resource("workitem.2", func() {
+	BasePath("/workitems.2")
```

Comment:
Should we name these based on 'feature' instead of a number? e.g. /workitem_paged ?